lib/uknetdev/netdev.c

```diff
@@ -139,40 +107,63 @@ static struct uk_netdev_einfo_overwrites *_alloc_einfo(struct uk_alloc *a,
 
 	/*
 	 * Parse IPv4 parameters
-	 * NOTE: `pick_arg` automatically returns NULL if an arguments is
-	 *       empty or does not exists.
+	 * NOTE: `uk_nextarg` automatically returns NULL if an arguments
+	 *       does not exists.
 	 */
-	_einfo->ip4.cidr     = pick_arg(&ipv4_conf[netdev_id], ':');
-	_einfo->ip4.gw       = pick_arg(&ipv4_conf[netdev_id], ':');
-	_einfo->ip4.dns0     = pick_arg(&ipv4_conf[netdev_id], ':');
-	_einfo->ip4.dns1     = pick_arg(&ipv4_conf[netdev_id], ':');
-	_einfo->ip4.hostname = pick_arg(&ipv4_conf[netdev_id], ':');
-	_einfo->ip4.domain   = pick_arg(&ipv4_conf[netdev_id], ':');
+	_einfo->ip4.cidr     = uk_nextarg(&ipv4_conf[netdev_id], ':');
```

Comment:
Let's maintain consistency with the spacing around `=`? :thinking: This way, if someone in the future adds another longer named parameter they wouldn't have to re-align. Could be just my subjective opinion though. :man_shrugging:

Reply:
You want a single space, nothing aligned? I don't mind actually.